### PR TITLE
add CvssV4VulnAssessmentRelationship.md and CvssSeverityType.md

### PR DESCRIPTION
**Pull request by jeff-schutt**
Initial draft of changes to support CVSS version 4

CC: @rnjudge @tsteenbe @goneall

### CODE REVIEW

File: model/Security/Classes/CvssV4VulnAssessmentRelationship.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Metadata
+
+- name: CvssV4VulnAssessmentRelationship
+- SubclassOf: VulnAssessmentRelationship
+- Instantiability: Concrete
+
+## Properties
+
+- Score
```
Comment (goneall) [inline model/Security/Classes/CvssV4VulnAssessmentRelationship.md:70]:
```suggestion
- score
```

File: model/Security/Classes/CvssV4VulnAssessmentRelationship.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  - type: xsd:decimal
+  - minCount: 1
+  - maxCount: 1
+- Severity
```
Comment (goneall) [inline model/Security/Classes/CvssV4VulnAssessmentRelationship.md:74]:
```suggestion
- severity
```

Properties should start with lower case

File: model/Security/Classes/CvssV4VulnAssessmentRelationship.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+**Syntax**
+
+```json
+{
+  "@type": "CvssV4VulnAssessmentRelationship",
+  "@id": "urn:spdx.dev:cvssv4-cve-2020-28498",
+  "relationshipType": "hasAssessmentFor",
+  "severity": "medium",
+  "score": 6.8,
+  "vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:C/C:H/I:N/A:N",
```
Comment (goneall) [inline model/Security/Classes/CvssV4VulnAssessmentRelationship.md:28]: From a quick glance, should this start with "CVSS:4"?

**Review by goneall — CHANGES_REQUESTED** [review]
Just a few things I noticed.

Also, as a reminder, we'll need to update the[ severity property range](https://github.com/spdx/spdx-3-model/blob/f5976aee78e3c1bc2024eca8631a7d20281ffb6c/model/Security/Properties/severity.md?plain=1#L18) to be `CvssSeverityType`, the [CvssV2VulnAssessmentRelationship severity type](https://github.com/spdx/spdx-3-model/blob/f5976aee78e3c1bc2024eca8631a7d20281ffb6c/model/Security/Classes/CvssV2VulnAssessmentRelationship.md?plain=1#L75C5-L75C5) to `CvssSeverityType` and the [CvssV3VulnAssessmentRelationship severity type](https://github.com/spdx/spdx-3-model/blob/f5976aee78e3c1bc2024eca8631a7d20281ffb6c/model/Security/Classes/CvssV3VulnAssessmentRelationship.md?plain=1#L78) to `CvssSeverityType`

File: model/Security/Classes/CvssV4VulnAssessmentRelationship.md (hunk 3 above)
Comment (jeff-schutt) [reply model/Security/Classes/CvssV4VulnAssessmentRelationship.md]: Addressed in latest commit, thanks!

**Review by goneall — APPROVED** [review]
LGTM

**Review by kestewart — APPROVED** [review]
Looks good to me.

File: model/Security/Classes/CvssV4VulnAssessmentRelationship.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      "@type": "ExternalRef",
+      "externalRefType": "securityAdvisory",
+      "locator": "https://logging.apache.org/log4j/2.x/security.html"
+    },
+    {
+      "@type": "ExternalRef",
+      "externalRefType": "securityOther",
+      "locator": "    https://www.first.org/cvss/v4.0/examples#Apache-log4j-JNDI-Command-Execution-log4shell-Vulnerability-CVE-2021-44228"
+    },
+    }
```
Comment (jeff-schutt) [inline model/Security/Classes/CvssV4VulnAssessmentRelationship.md:47]:
```suggestion
```

**Review by jeff-schutt — COMMENTED** [review]
Minor typo

**Pull request merged**